src/app/components/hamburger/hamburger.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="hamburger" (click)="onClickHamburger()">
+  <img
+    src="assets\images\menu.svg.svg"
+    alt="menu"
+    *ngIf="!isOpened"
+  >
+</div>
+<div class="backdrop"
+  *ngIf="isOpened"
+  (click)="onClickHamburger()"
+>
+  <div class="menu"
+  *ngIf="isOpened">
+    <img
+      src="assets\images\close_icon.svg"
+      alt="menu"

```

Comment:
altを指定する場合、画像の役割が分かるようにしましょう
```suggestion
      alt="close menu"
```